group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetMetadataManager.java

```diff
@@ -862,6 +863,7 @@ public void replay(
             }
 
             if (producerId == RecordBatch.NO_PRODUCER_ID) {
+                log.debug("Replaying offset commit with producer id {}, key {}, value {}", producerId, key, value);
```

Comment:
Do we care to print out the producer ID if it is "no producer id"?

Reply:
No, we can remove it.